### ISSUE TITLE
> Rails 6: Use patched version of Foreman gem

### ISSUE THREAD

**Issue by alaxalves** [publiclab/mapknitter#989]
Since we got https://github.com/publiclab/mapknitter/pull/956 merged into main, bundling the dependencies will cause an error regarding the `thor` gem version mismatch. I have already solved this in https://github.com/publiclab/mapknitter/pull/824. I'll submit a PR fixing this asap, we we shold merge it asap.

**Comment by gauravano** [comment]
> Since we got #956 merged into main, bundling the dependencies will cause an error regarding the thor gem version mismatch.

Where are we getting this version mismatch error?? I am not able to reproduce.

**Comment by alaxalves** [comment]
> > Since we got #956 merged into main, bundling the dependencies will cause an error regarding the thor gem version mismatch.
> 
> Where are we getting this version mismatch error?? I am not able to reproduce.

```
When doing a fresh MK installation, something like this pops up:
Bundler could not find compatible versions for gem "thor":
  In snapshot (Gemfile.lock):
    thor (= 0.20.3)

  In Gemfile:
    foreman (~> 0.85.0) was resolved to 0.85.0, which depends on
      thor (~> 0.19.1)

    rspec-rails was resolved to 3.8.2, which depends on
      railties (>= 3.0) was resolved to 6.0.0, which depends on
        thor (>= 0.20.3, < 2.0)
```

Better described here as well: https://github.com/ddollar/foreman/issues/735

**Comment by alaxalves** [comment]
As discussed with @gauravano I'm closing this and https://github.com/publiclab/mapknitter/pull/990 in favor of #979
